src/pages/sykmeldt/index.tsx

```diff
@@ -7,10 +7,30 @@ import { beskyttetSideUtenProps } from "../../auth/beskyttetSide";
 import SykmeldtSide from "../../components/blocks/wrappers/SykmeldtSide";
 import OppfolgingsdialogerInfoPersonvern from "../../components/blocks/infoboks/OppfolgingsdialogerInfoPersonvern";
 import VideoPanel from "../../components/blocks/video/VideoPanel";
+import { useNarmesteLedereSM } from "../../api/queries/sykmeldt/narmesteLedereQueriesSM";
 
 const Home: NextPage = () => {
   const oppfolgingsplaner = useOppfolgingsplanerSM();
   const sykmeldinger = useSykmeldingerSM();
+  const narmesteLedere = useNarmesteLedereSM();
+
+  const PageContent = () => {
+    if (
+      oppfolgingsplaner.isSuccess &&
+      sykmeldinger.isSuccess &&
+      narmesteLedere.isSuccess
+    ) {
+      return (
+        <OppfolgingsplanContent
+          oppfolgingsplaner={oppfolgingsplaner.data}
+          sykmeldinger={sykmeldinger.data}
+          narmesteLedere={narmesteLedere.data}
+        />
+      );
+    }
+
+    return null;
+  };
```

Comment:
@luantr Kan dette lage samme problem med rerendering som vi så for modalen?

Reply:
Ja, stemmer! Ved deklarering av komponenter i en annen komponent så vil re-render av parent trigge en mount/un-mount.`PageConent` burde flyttes ut av komponenten

Reply:
Flytter den ut :)